server/src/main/java/org/elasticsearch/common/blobstore/BlobPurpose.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.common.blobstore;
+
+public enum BlobPurpose {
```

Comment:
Sorry for nitpicking about naming here but I don't like the new name `BlobPurpose`. We're not distinguishing _blobs_ by purpose, it's the _operations_ on them.

Reply:
OK I reverted it back to `OperationPurpose`.